mysql-test/main/func_str.test

```diff
@@ -2193,3 +2193,51 @@ SELECT GROUP_CONCAT( UpdateXML( '<a>new year</a>', '/a', '2019-01-01 00:00:00' )
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
+--echo #
+--echo # MDEV-25704 Function random_bytes
+--echo #
+
+--source include/have_sequence.inc
+
+create table t1 as select random_bytes(100);
+select seq from seq_1_to_1024 where not (random_bytes(seq) <=> random_bytes(seq)) and length(random_bytes(seq)) = seq;
```

Comment:
this is questionable. As random bytes are, well, random, there is no guarantee that two subsequent calls will return different bytes. Particularly for `random_bytes(1) <=> random_bytes(1)`, which, I expect, will fail about every 256th run.

Reply:
That's true. I'm in the process of fixing it. Commit will be pushed soon.